config/target.in

```diff
@@ -62,6 +63,9 @@ config MULTILIB
       NOTE: The multilib feature in crosstool-NG is not well-tested.
             Use at your own risk, and report success and/or failure.
 
+config MULTILIB_WITH_ARM_32
+    bool
+
```

Comment:
I'm still not wild about adding architecture specific options to target.in. I think I'm erring on the side of capturing this in arch/arm.in or make the option something like `ARCH_MULTILIB_EXCLUSIVE` and have arm.in `select` it.